userguide/system.rst

```diff
@@ -1589,6 +1589,11 @@ Dropbox, FTP, Google Cloud Storage, Google Drive, HTTP, Hubic, Mega,
 Microsoft Azure Blob Storage, Microsoft OneDrive, pCloud, SFTP, WebDAV,
 and Yandex are supported.
 
+.. warning:: Cloud Credentials are stored in encrypted form. To be able
+   to restore Cloud Credentials from a
+   :ref:`saved configuration<General>`, "Export Password Secret Seed"
```

Comment:
There should be a space between the label and the link, although it might work without that.
```suggestion
   :ref:`saved configuration <General>`, "Export Password Secret Seed"
```